client/src_analyzer/Worker_Analyze.re

```diff
@@ -213,4 +213,12 @@ module Make = (ESig: Worker_Evaluator.EvaluatorSig) => {
            });
       result;
     };
+
+  let executeMany:
+    Belt.Map.String.t(string) => Belt.Map.String.t(list(blockData)) =
+    codeMap => {
```

Comment:
This should be an explicit uncurried function. Bucklescript and ES6 proxy doesn't play nice with each others.

(. codeMap) =>